### PR TITLE
Add `cooking-pod` icon

### PR DESCRIPTION
**Pull request by guillermo-angeles**

### CODE REVIEW

File: icons/cooking-pot.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+    stroke-width="2"
+    stroke-linecap="round"
+    stroke-linejoin="round"
+>
+    <line x1="2" y1="11" x2="22" y2="11"/>
+    <polyline points="10.3 5.42 9.82 3.48 13.7 2.51 14.18 4.45"/>
```
Comment (jguddas) [inline icons/cooking-pot.svg:14]: There needs to be a whole that stays visible even when you have 3.9px stroke width in the handle, this is ugly but legal icon (47)

Comment (jguddas) [reply icons/cooking-pot.svg:14]: Very nice icon btw, love me all food related icons that have a bit of swoosh.

**Comment by ericfennis** [comment]
Maybe we can try to explore different designs for the lid?

**Comment by ericfennis** [comment]
What about giving it a more obvious handle?
image

**Comment by jguddas** [comment]
> What about giving it a more obvious handle? image

I'm okay with that, feel free to create a PR.

File: icons/cooking-pot.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <path d="M20,11v8c0,1.1-.9,2-2,2H6c-1.1,0-2-.9-2-2V11"/>
+    <line x1="4" y1="7" x2="20" y2="3"/>
```
Comment (ericfennis) [inline icons/cooking-pot.svg:16]:
```suggestion
        <path d="M2 12h20" />
        <path d="M20 12v8a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2v-8" />
        <path d="m4 8 16-4" />
        <path d="m8.86 6.78-.45-1.81a2 2 0 0 1 1.45-2.43l1.94-.48a2 2 0 0 1 2.43 1.46l.45 1.8" />
```

Comment (ericfennis) [reply icons/cooking-pot.svg:16]: @guillermo-angeles Can you apply this suggestion?

**Comment by ericfennis** [comment]
Changes applied in #1737

**Pull request closed without merging**